corehq/apps/users/bulkupload.py

```diff
@@ -487,12 +487,14 @@ def is_password(password):
                     if language:
                         user.language = language
                     if email:
-                        email_validator = EmailValidator()
                         try:
-                            email_validator(email)
-                            user.email = email
+                            validate_email(email)
                         except ValidationError:
                             raise UserUploadError(_("User has an invalid email address"))
+
+                        if not email.islower():
+                            raise UserUploadError(_("Email address must be lower case: {}".format(email)))
```

Comment:
I know the ticket asks for a better error message... but what is the reason we don't allow caps in email addresses? 
Can we not just force the email address to be lower case, and not make this an error condition?